src/main/scala/beam/router/skim/CsvSkimReader.scala

```diff
@@ -21,18 +22,18 @@ import scala.util.control.NonFatal
   */
 class CsvSkimReader(
   val aggregatedSkimsFilePath: String,
-  fromCsv: Map[String, String] => (AbstractSkimmerKey, AbstractSkimmerInternal),
+  fromCsv: scala.collection.Map[String, String] => (AbstractSkimmerKey, AbstractSkimmerInternal),
   logger: Logger
 ) {
 
-  val header: Array[String] = initHeader
+  val header: Array[String] = initHeader()
 
   def readAggregatedSkims: immutable.Map[AbstractSkimmerKey, AbstractSkimmerInternal] = {
     var res = Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
     val csvParser: CsvParser = getCsvParser
```

Comment:
csvReader is created twice. May be move code from `initHeader()` here?

Reply:
@crixal as per discussion, let's repair it in the separate PR